exercises/isogram/isogram_test.rb

```diff
@@ -7,55 +7,55 @@ class IsogramTest < Minitest::Test
   def test_empty_string
     # skip
     string = ""
-    assert Isogram.is_isogram?(string)
+    assert Isogram.isogram?(string), "Expected 'true', '' is an isogram"
```

Comment:
No definitely, I get what you're saying! I guess I was just going off the triangle exercise test case more or less. If we decide to change it on here I can also go back and change it here later https://github.com/exercism/xruby/blob/master/exercises/triangle/triangle_test.rb. 😄